frontend/src/components/ui/WorkspaceTabs.jsx

```diff
@@ -2,11 +2,16 @@ import { useEffect, useState } from "react";
 import { Tabs, TabList, Tab, TabPanels, TabPanel } from "@carbon/react";
 import { workspaceAtom } from "@/atoms/pipelineAtom";
 import { useImmerAtom } from "jotai-immer";
+import { useAtom } from "jotai";
+import { drawflowEditorAtom } from "@/atoms/drawflowAtom";
 
 export default function WorkspaceTabs() {
   const [workspace, setWorkspace] = useImmerAtom(workspaceAtom);
   const [renderedTabs, setRenderedTabs] = useState([]);
   const [selectedIndex, setSelectedIndex] = useState(0);
+  const [zoom_levels, setZoomLevels] = useState({});
+
+  const [editor] = useAtom(drawflowEditorAtom); // Accessing the Drawflow editor instance from a global state
```

Comment:
I think this code is clear enough and we don't need this comment

Reply:
removed the comment.